scipy/stats/_distn_infrastructure.py

```diff
@@ -1408,11 +1412,13 @@ class rv_continuous(rv_generic):
         This string is used as the last part of the docstring returned when a
         subclass has no docstring of its own. Note: `extradoc` exists for
         backwards compatibility, do not use for new subclasses.
-    seed : None or int or ``numpy.random.RandomState`` instance, optional
-        This parameter defines the RandomState object to use for drawing
-        random variates.
-        If None (or np.random), the global np.random state is used.
-        If integer, it is used to seed the local RandomState instance.
+    seed : {None, int, `~np.random.RandomState`, `~np.random.Generator`}, optional
```

Comment:
Do we want to add anything in docstrings anywhere about the numpy version needed for the Generator? I have no strong feelins on yes/no. I'm not sure what the typical user would know in this case.

Reply:
The user will probably realise quite quickly that it won't work if they can't create a Generator with their own numpy. I don't know where that information would go though, there are a lot of places that use random numbers.

Reply:
yeah, and because it is so far across things, the release notes are probably the best place for this sort of thing anyway, so no need for changes in dcostring on this item.